basket/news/migrations/0026_newsletter_is_waitlist.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 3.2.16 on 2023-01-25 15:02
+
+from django.db import migrations, models
+
+
+def set_waitlist_field(apps, schema_editor):
+    """
+    Instead of hard-coding which newsletters are waitlists,
+    we now set a flag.
+    """
+    Newsletter = apps.get_model("news", "Newsletter")
+    for newsletter in Newsletter.objects.all():
+        is_waitlist = newsletter.slug.endswith("-waitlist") or newsletter.slug in {
+            "relay",
+            "fpn",
```

Comment:
Should this be `"vpn"`? Based on the above conditionals we check for the `"vpn"` newsletter and change it to `"guardian-vpn-waitlist"` but I've only seen `"fpn"` used as part of the extra data for the waitlist.

Reply:
You are right, this conditional is superfluous. Looking at the list at https://basket.mozilla.org/news/ all waitlists slugs already end with `-waitlist`